lib/request.js

```diff
@@ -2,11 +2,24 @@
  * Parse params found in path of the request URL
  *
  * @param {Request} request
- * @returns {{proofSetId?: string, pieceCid?: string}}
+ * @returns {{clientWalletAddress?: string, pieceCid?: string}}
```

Comment:
```suggestion
 * @returns {{clientWalletAddress?: string, pieceCid?: string, error?: string}}
```

There's `error` field missing in the return type.